pkg/harvester/dialog/HarvesterMigrationDialog.vue

```diff
@@ -23,10 +23,33 @@ export default {
     }
   },
 
+  async fetch() {
+    try {
+      if (!this.actionResource.hasAction('findMigratableNodes')) {
```

Comment:
We should not display the `Migration` menu without the `findMigratableNodes` action.